CHANGELOG.md

```diff
@@ -8,6 +8,8 @@ and this project adheres to [Semantic Versioning](http://semver.org/).
 ## [Unreleased]
 
 - Improves caching strategy and use new scope convention to declare this a core Heroku CLI plugin. 
+- Added command `pipelines:connect` to connect an existing pipeline to a Github repo.
+- Added command `pipelines:reviewapps` to enable|disable review apps with an existing pipeline.
```

Comment:
I'm wondering whether the namespace should be `reviewapps` so it can be extensible to command in which won't involve a pipeline (if/when) a review app could live out of a pipeline, and it also has the same design as we have for Heroku CI for example (`heroku ci:config -p pipeline`, or `heroku ci:enable -p pipeline` which doesn't exist yet).

This could be done in this `heroku-pipelines` plugin directly. Similarly to other commands like `apps`, `members`, etc. Here's an [example](https://github.com/heroku/heroku-orgs/blob/master/index.js#L18-L34).

The commands that would replace what you're proposing with `pipelines:reviewapps` would be:

```bash
# To enable review apps in a pipeline (when app not specified)
$ heroku reviewapps:enable -p pipeline
(prompt) Choose the parent application:
[ ] Existing App 1
[x] Existing App 2

# To enable review apps in a pipeline (specifying app)
$ heroku reviewapps:enable -p pipeline -a app

# To disable review apps in a pipeline:
$ heroku reviewapps:disable -p pipeline

# To set a parent app
$ heroku reviewapps:set -a myapp -p pipeline

# To enable autodeploy
$ heroku reviewapps:enable --autodeploy -p pipeline

# To disable autodeploy
$ heroku reviewapps:disable --autodeploy -p pipeline

# To enable autodestroy
$ heroku reviewapps:enable --autodeploy -p pipeline

# To disable autodestroy
$ heroku reviewapps:disable --autodeploy -p pipeline

# To enable review apps, autodeploy and autodestroy 
$ heroku reviewapps:enable --autodeploy --autodestroy -p pipeline
```

@wadewegner What do you think?

Reply:
cc @heroku/cli see my comment above in case you have an opinion about the design.

Reply:
I'm happy to make this change. @joshwlewis mentioned something about `heroku reviewapps` and if this fits with his design 👍. I'll wait to hear confirmation from you or others.

Reply:
@raulb I've made these changes. Note: I didn't add `heroku reviewapps:set`. As I went over and over this, I realized it wasn't necessary.

Reply:
+1 for @raulb's suggestion